common/web/input-processor/src/text/inputProcessor.ts

```diff
@@ -314,9 +313,9 @@ export default class InputProcessor {
 
           let mock = Mock.from(windowedMock, false);
 
-          let altKey = activeLayout.getLayer(keyEvent.kbdLayer).getKey(pair.keyId);
+          const altKey = pair.keySpec;
           if(!altKey) {
-            console.warn("Potential fat-finger key could not be found in layer!");
+            console.warn("Internal error:  failed to properly filter set of keys for corrections");
```

Comment:
This obviously is connected to this ongoing Sentry error:  https://keyman.sentry.io/share/issue/977c8e43dc1b40ddb7f6948252a6118b/

Note that the new pattern directly uses the object that was previously retrieved by lookup.  There will no longer be an inability to _find_ the object.  

Of course, that doesn't solve the _whole_ problem - whatever had happened to the layer, and how whatever that is should be handled in relation to fat-fingering - but this should help us get closer to finding the root of the issue, even then.